tachyon/zk/base/query_unittest.cc

```diff
@@ -0,0 +1,58 @@
+#include <tuple>
+
+#include "gtest/gtest.h"
+
+#include "tachyon/crypto/commitments/kzg/kzg_commitment_scheme.h"
+#include "tachyon/math/elliptic_curves/bn/bn254/g1.h"
+#include "tachyon/math/elliptic_curves/bn/bn254/g2.h"
+#include "tachyon/zk/base/polynomial_pointer.h"
```

Comment:
Please remove this.

Reply:
These are necessary to determine the `PCS` type.